### DISCUSSION TITLE
Multi language support for blocks

### DISCUSSION THREAD

**Discussion by ivl04** [Z3d0X/filament-fabricator#157] [Ideas]
Implementing multi-language support would involve integrating language localization capabilities into Filament Fabricator. This would allow users to select their preferred language from a list of supported languages, with the interface and any text displayed dynamically adjusting to the chosen language.

as we can use spatie's translable to add multi lang data if user is using fabricator for storing blocks it should be like 
```
[{"type":"card","data":{"image":"http:\/\/127.0.0.1:8000\/storage\/01HWQGNGZMDTVBS4PZYVTAX3CE.png","title":{"en":"ddd"},"tooltip":{"en":"ddd"},"description":"<p>dddd<\/p>","count":"2"}}]
```

so that  localization will be easy

**Comment by barkoczi** [comment]
Hi Ivl04, 

have you implemented it?

**Comment by Voltra** [comment]
See #119 and #153
